.github/workflows/ohos.yml

```diff
@@ -182,7 +181,6 @@ jobs:
     # so in the beginning we will just do a best effort approach but ignore errors.
     continue-on-error: true
     runs-on: hos-runner
-    if: github.repository == 'servo/servo'
```

Comment:
I guess you removed these lines for testing with your fork, right? Could you link a github actions run from your fork?
And lets leave this comment open to not forget reverting this before merging